django_ses/management/commands/get_ses_statistics.py

```diff
@@ -29,13 +26,10 @@ class Command(BaseCommand):
 
     def handle(self, *args, **options):
 
-        connection = SESConnection(
+        connection = boto3.client(
+            'ses',
             aws_access_key_id=settings.ACCESS_KEY,
             aws_secret_access_key=settings.SECRET_KEY,
```

Comment:
I had to add these lines to make it work on my environment (copied from the other command `ses_email_address.py`), but I dont think this is related to the current on going work. My problem is, by default I use a region which does not have SES, thus I have to manually chose the ses region to work on.

```python
            region_name=settings.AWS_SES_REGION_NAME,
            endpoint_url=settings.AWS_SES_REGION_ENDPOINT_URL,
```

Reply:
@trecouvr Shall I add these two lines to the code?

Reply:
I think you should, these are the same lines as in the other parts of the code. It allows to specify the ses region.

I would say the best move would be to refactor all the boto.client calls present in the lib and in the commands into one function, but this could be done later.

Reply:
Pushed it. Thanks for the review. And yes, we definetely should refactor it at some point.